plugins/services/src/js/structs/Framework.js

```diff
@@ -20,6 +20,32 @@ module.exports = class Framework extends Application {
     return this.get("slave_ids");
   }
 
+  getFullId() {
```

Comment:
`#getFullId` is a little misleading here because it's actually the full Id but without the leading `/`. I think we should change this method to `#getId` so it overrides the super method and instead of pruning the leading `/` we should make sure it is always present.